rider-fsharp/build.gradle.kts

```diff
@@ -231,8 +231,9 @@ configure<RdGenExtension> {
 
 tasks {
     withType<PrepareSandboxTask> {
-        var files = libFiles + pluginFiles.map { "$it.dll" } + pluginFiles.map { "$it.pdb" } + typeProvidersFiles + fantomasFiles
+        var files = libFiles + pluginFiles.map { "$it.dll" } + pluginFiles.map { "$it.pdb" } + typeProvidersFiles
         files = files.map { "$resharperPluginPath/src/$it" }
+        val fantomasFiles = fantomasFiles.map { "$resharperPluginPath/src/$it" }
```

Comment:
Will `{ "../fantomas/$it" }` work?

Reply:
No, because Fantomas.Host is contained in ReSharper.FSharp

Reply:
Ah, that's the from path, my bad.
Thanks!